src/test/java/com/wikia/webdriver/common/core/elemnt/Wait.java

```diff
@@ -416,4 +412,20 @@ public void forXMilliseconds(int time) {
       PageObjectLogging.log("Wait.forXMilliseconds", e, false);
     }
   }
+
+  private WebElement forElementPresent(WebDriverWait wait, By by, boolean failOnTimeout) {
```

Comment:
Could you use https://github.com/Wikia/selenium-tests/blob/ADEN-3097-extend-evolve-params/src/test/java/com/wikia/webdriver/common/core/elemnt/Wait.java#L65 ?